static/resources/harness.js

```diff
@@ -131,6 +131,39 @@
     return result;
   }
 
+  function processValue(value, data, test, result) {
+    if (value && typeof value === 'object' && 'result' in value) {
```

Comment:
This is moved code, not new, but what's this for? Are some tests returning `{ result: true }` instead of just `true`?

Reply:
Not currently, but this will allow us to do so.  This is useful for when we wish to add a custom message to test results.